kt/entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/generator/typehint/coretypes/JvmCoreTypeTypeHintGenerator.kt

```diff
@@ -0,0 +1,23 @@
+package godot.entrygenerator.generator.typehint.coretypes
+
+import com.squareup.kotlinpoet.ClassName
+import godot.entrygenerator.exceptions.WrongAnnotationUsageException
+import godot.entrygenerator.generator.typehint.PropertyTypeHintGenerator
+import godot.entrygenerator.model.ColorNoAlphaHintAnnotation
+import godot.entrygenerator.model.RegisteredProperty
+
+class JvmCoreTypeTypeHintGenerator(
+    private val registeredProperty: RegisteredProperty
+) : PropertyTypeHintGenerator(registeredProperty) {
+    override fun getPropertyTypeHint(): ClassName {
+        return when (propertyHintAnnotation) {
+            ColorNoAlphaHintAnnotation -> ClassName("godot.core.PropertyHint", "COLOR_NO_ALPHA")
+            //TODO: implement ImageCompressLossy
```

Comment:
TODOs are normal, or implems are forgotten ?

Reply:
They are "normal". If you look at https://github.com/utopia-rise/godot-kotlin-jvm/blob/master/kt/godot-library/src/main/kotlin/godot/annotation/propertyHintAnnotations.kt All those annotations are commented. We previously decided that we do not yet implement hint annotations from which we do not know the purpose or how they function.
So i kept them as we had them before. With comments and todos.
But i can of course also completely remove them for now if you prefer